Libraries/Connect/Internal/Interceptors/GRPCWebInterceptor.swift

```diff
@@ -61,12 +61,38 @@ extension GRPCWebInterceptor: UnaryInterceptor {
                 response.headers,
                 trailers: response.trailers
             )
+            if grpcCode != .ok || connectError != nil {
```

Comment:
What about if `grpcCode == .ok && connectError != nil`? That would fall into the first case. We should probably change `grpcCode` to be "unknown" in that case. WDYT?

Reply:
Makes sense to me, will update

Reply:
Updated